nox/__init__.py

```diff
@@ -14,6 +14,7 @@
 
 from __future__ import annotations
 
+from nox import toml
```

Comment:
Should we support `nox.toml.load` out of the box, or should it require `import nox.toml`? The latter is a bit more efficient, since you only load it if you need it (but we could make this lazy). `rich` and `textual` go this way.

Reply:
I'd go with convenience over efficiency here, I don't think the extra import will hurt anyone in the context of a Nox run.